src/ripple/nodestore/backend/RocksDBFactory.cpp

```diff
@@ -352,11 +358,6 @@ class RocksDBBackend
     {
         storeBatch (batch);
     }
-
-    void
-    verify() override
```

Comment:
Just checking that losing this `verify() override` is correct.  We want the behavior that comes from the base class?  I'll assume so...